src/utils/Ponyfills.js

```diff
@@ -0,0 +1,15 @@
+// This file is intented to have multiple exports
+// eslint-disable-next-line import/prefer-default-export
+export function convertUint8ArrayToString(array) {
+    if (window.TextDecoder !== undefined) {
+        return new TextDecoder().decode(array);
+    }
+
+    // Avoid the String.fromCharCode.apply(null, array) shortcut, which
+    // throws a "maximum call stack size exceeded" error for large arrays.
+    var s = '';
+    for (var i = 0, il = array.length; i < il; i++) {
+        s += String.fromCharCode(array[i]);
```

Comment:
Ok not using https://www.npmjs.com/package/text-encoding-shim because it uses `escape`, which is deprecated. I'm going to test https://github.com/arv/text-encoding-utf-8